api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -1,6 +1,21 @@
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <service>api</service>
   <runtime>java8</runtime>
+
+  <!--
+  The GAE java8 runtime restricts certain JVM APIs. The java11 runtime has fewer restrictions.
+  Google provides GAE-specific libraries to work around these restrictions, but using these
+  libraries on the java11 runtime throws a:
+    com.google.apphosting.api.ApiProxy$FeatureNotEnabledException
+
+  See: https://cloud.google.com/appengine/docs/standard/java-gen2/services/access
```

Comment:
A link here would be informative too https://cloud.google.com/appengine/docs/standard/bundled-services-overview

Reply:
Good one.